docs/components/InputFile/Web.stories.tsx

```diff
@@ -1,13 +1,18 @@
 import React, { useCallback, useState } from "react";
 import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Card } from "@jobber/components/Card";
```

Comment:
Since the button variation is no longer deprecated I think it would be worth having an example of a `Button` variant of the InputFile with customization like icons on the button.

You could probably update the CustomButton story and have a Custom dropzone button story